### PR TITLE
merge compatible into develop

### PR DESCRIPTION
**Pull request by deepthiskumar**
Checklist:

- [ ] Modified the current draft of release notes with details on what is completed or incomplete within this project
- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them

* Closes #0000

### CODE REVIEW

**Comment by deepthiskumar** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!ci-build-me

File: src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml [hunk 1]
```diff
@@ -963,39 +963,45 @@ let setup_daemon logger =
...
                   ]
-                "Long async cycle, $long_async_cycle seconds" ;
+                "Long async cycle, $long_async_cycle seconds, $monitors" ;
               Mina_metrics.(
                 Runtime.Long_async_histogram.observe Runtime.long_async_cycle
                   secs) ) ;
           Stream.iter Async_kernel.Async_kernel_scheduler.long_jobs_with_context
             ~f:(fun (context, span) ->
               let secs = Time_ns.Span.to_sec span in
+              let monitor_infos = get_monitor_infos context.monitor in
```
Comment (rbonichon) [inline src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml:1000]: Can this `monitor_infos` be different from the one on L987. If so, a comment could help (not blocking ;-) if not, just reuse the old values.

**Review by rbonichon — APPROVED** [review]
LGTM

File: src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml (hunk 1 above)
Comment (mrmr1993) [reply src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml:1000]: Yes, it's explicitly different.

This PR is a merge, it's best to add these comments to the original PR / against compatible, and to direct fixes there for the next merge IMO.

**Review by mrmr1993 — APPROVED** [review]

**Comment by deepthiskumar** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!ci-build-me

**Review by lk86 — APPROVED** [review]

**Comment by deepthiskumar** [comment]
!ci-build-me

**Comment by mrmr1993** [comment]
This is super out-of-date. Closing.

**Pull request closed without merging**